### DISCUSSION TITLE
Warning: File ignored because of a matching ignore pattern. This makes no sense

### DISCUSSION THREAD

**Discussion by Jakobud** [eslint/eslint#14261] [Q&A]
In my `eslintrc.js` I have the following ignore patterns:

```
...
ignorePatterns: [
  'bin',
  'src/website-2/**/*.js',
  'test',
  'build_util'
],
...
```

When linting I run this:

```
eslint "src/**/*.js"
```

And I end up seeing linting errors for the `src/website-2` directory JS files. What is it about my ignorePatterns that is incorrect?

**Comment by nzakas** [comment]
It’s hard to know without seeing the full directory structure. You can try using the —debug flag to see if there’s more info to help.
